backend/backend/settings.py

```diff
@@ -153,9 +152,9 @@
 EMAIL_PORT = 587
 EMAIL_USE_TLS = True
 
-CORS_ORIGIN_ALLOW_ALL = bool(os.getenv("ALLOW_CORS", False))
+
+CORS_ORIGIN_ALLOW_ALL = bool(os.getenv("ALLOW_CORS", True))
```

Comment:
This should default to False, not True; this is an insecure setting that we should only be using for development.

Reply:
When I set it to false, I am unable to fetch any data from the backend ```Access to fetch at 'http://localhost:8000/events/' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.```

Reply:
When false, I am unable to fetch any data from the backend.

I get the following error message:

`Access to fetch at 'http://localhost:8000/events/' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`

Reply:
Are you setting the environment variable "ALLOW_CORS" to True in your .env file?